public/src/node.js

```diff
@@ -0,0 +1,146 @@
+// # Analyzing Data with Node
+//
+// As mentioned in the [introduction](getting_started.html), this guide is mostly geared for client-side data analysis, but with a few augmentations, the same tools can be readily used server-side with [Node](https://nodejs.org/).
+//
+// And the data is too large, this might in fact be your only option if you want to use JavaScript for your data analysis. None of your users will wait 5 minutes on your site while a giant file is loaded up, no matter how cool the analysis.
```

Comment:
"None of your users will wait 5 minutes on your site while a giant file is loaded up, no matter how cool the analysis."

I know what you're saying here: If your data is too large, processing it client side might incur too long of a wait. However, you can use node.js to build servers, and then try to do the same data analysis INSIDE YOUR CALLBACKS. Which, of course, is a bad idea. I'm not sure exactly how to rephrase it, but I think pointing that out might be worthwhile. Suggestion:

"Trying to deal with large data in the browser might result in your users having to wait for a long time. No user will wait for 5 minutes with a frozen browser, no matter how cool the analysis might be." <- More browser specific?